Code/Engine/RendererCore/AnimationSystem/AnimGraph/AnimGraphNode.h

```diff
@@ -59,26 +59,6 @@ class EZ_RENDERERCORE_DLL ezAnimGraphNode : public ezReflectedClass
 //////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////
 
-struct ezComparisonOperator

```

Comment:
Ok I see you moved this from here to the blackboard, but maybe there is an even better place.